ocp4/profiles/moderate.profile

```diff
@@ -131,6 +132,7 @@ selections:
     - grub2_vsyscall_argument.role=unscored
     - grub2_vsyscall_argument.severity=info
     - grub2_pti_argument
+    - sysctl_user_max_user_namespaces
```

Comment:
Do we really want to disable user namespaces for a platform that does nothing but run containers?